src/pages/NewWallet/CreateWalletPage.tsx

```diff
@@ -52,7 +52,7 @@ const CreateWalletPage = ({ isRestoring = false }: { isRestoring?: boolean }) =>
   const onUpdatePassword = (password: string): void => {
     let passwordError = ''
 
-    if (password.length) {
+    if (password.length && !import.meta.env.DEV) {
```

Comment:
I thought it's a good idea to disable this (and the following) checks when in dev mode. WDYT?

Reply:
I think it is a good idea to speed up the testing process when in dev mode, definitely.
But I'm a bit afraid that this may hide some bugs.

Maybe we add a switch somewhere (through a keyboard shortcut?) to toggle dev mode? This could only be done if `import.meta.env.DEV` exists of course.

Reply:
Here's a demo!

Cmd+d enables the GOD MOOOOODE 💪 

Of course, only if we are in the dev environment.

As a protection mechanism, even if the dev mode is enabled, I still check that we are in the dev environment before disabling that checks I want to disable. See commit: 37afa924

I've also added a VERY CLEAR indication that we are in god mode so that we don't forget we're in it!


https://user-images.githubusercontent.com/1579899/223966631-7006dfb2-4c93-4ffb-8073-30b3bef6313c.mp4

Reply:
I also tried to sprinkle some fun into our code base, if you don't like it I can remove it :)
✨

Reply:
Love it 😻  💯 :godmode: 

Makes me think about eastereggs... After v2.0 + mobile are released, we may add a little something somewhere related to calzone... :)

Reply:
Lol, I didn't know about this emoji :godmode: